web/src/containers/Login.js

```diff
@@ -40,7 +46,7 @@ class Login extends Component {
       return <Redirect to="/" />;
     }
 
-    if (showConsent) {
+    if (showConsent && !hasEverLoggedOn) {
```

Comment:
_ever_? 😆

Reply:
***EVER***. We've got some next-level tracking built into this app, Heath-rah.  :trollface: